Firestore/Example/Tests/Local/FSTRemoteDocumentCacheTests.mm

```diff
@@ -52,14 +51,18 @@ - (void)setUp {
 - (void)testReadDocumentNotInCache {
   if (!self.remoteDocumentCache) return;
 
-  XCTAssertNil([self readEntryAtPath:kDocPath]);
+  self.persistence.run("testReadDocumentNotInCache", [&]() {
```

Comment:
FWIW as helpful as I find these actions in the logs produced by the production code I'm not sure these super-long names add that much to tests. If you're of the same mind and willing to do so feel free to cut these down to "testing" or something similarly short and sweet. It's also fine as-is.

Reply:
I think it's very marginally more useful than "testing", since it at least identifies where it's coming from, but requires a roughly similar amount of thought for test writers. So I'm inclined to keep them as is.